index.js

```diff
@@ -0,0 +1,70 @@
+import Book from './modules/book.js';
+import nevigateTo from './modules/navigate.js';
+import Store from './modules/store.js';
+import date from './modules/date.js';
+
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+
+    books.forEach((book) => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('.table-body');
+
+    const tableRow = document.createElement('tr');
+
+    tableRow.innerHTML = `
+           <td>${book.title}</td>
+           <td class= "td-by">  By ${book.author}</td>
+           <td ><a href="#" class="remove-btn">Remove</a></td>
+        `;
+
+    list.appendChild(tableRow);
+  }
```

Comment:
- [ ] [OPTIONAL] Please consider navigating to the book list after a book is successfully added. Alternatively, you can display notifications, `success` or `failure` after a book is added `successfully` or `not added` respectively.  Currently, after the `add` button is clicked, nothing happens, so it is confusing if it was successful or not.